client/profile-wizard/steps/store-details.js

```diff
@@ -167,67 +177,110 @@ class StoreDetails extends Component {
 		return (
 			<Fragment>
 				<H className="woocommerce-profile-wizard__header-title">
-					{ __( 'Where is your store based?', 'woocommerce-admin' ) }
+					{ __( 'Welcome to WooCommerce', 'woocommerce-admin' ) }
 				</H>
 				<H className="woocommerce-profile-wizard__header-subtitle">
 					{ __(
-						'This will help us configure your store and get you started quickly',
+						"Tell us about your store and we'll get you set up in no time",
```

Comment:
A question: do I need to do anything for translations in other languages here?

Reply:
No, its wrapped in `__()`, so the translations are handled elsewhere.